### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by SadiqZak**
Added Cover, Explore and Footer with responsiveness. Need to add Links and Routes accordingly.

### CODE REVIEW

File: src/Pages/Homepage/Homepage.jsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import React from "react";
+
+const Homepage = () => {
+  return (
+    <div>
+      <img
+        className="cover"
+        src={require("../../Assets/cover.jpg")}
```
Comment (Devansu-Yadav) [inline src/Pages/Homepage/Homepage.jsx:8]: I would suggest not using `require()` here. Instead, you can directly pass the Image src like this:
```
src="../../Assets/cover.jpg"
```

File: src/Pages/Homepage/Homepage.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+          <img
+            className="secondary-cover"
+            src={require("../../Assets/secondary-cover-1.jpg")}
+            alt="homepage"
+          />
+        </div>
+        <div>
+          <img
+            className="secondary-cover"
+            src={require("../../Assets/secondary-cover-2.jpg")}
```
Comment (Devansu-Yadav) [inline src/Pages/Homepage/Homepage.jsx:30]: Same thing here. Please avoid using `require()` here.

File: src/Components/Header/Logo/Logo.jsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const Logo=()=> {
+  return (
+    <div className="logo">
+      <span className="logo-icon material-icons">shopping_cart</span> PrimeCart
```
Comment (Devansu-Yadav) [inline src/Components/Header/Logo/Logo.jsx:6]: Material Icons can be added directly from the [@mui/icons-material](https://www.npmjs.com/package/@mui/icons-material) package. 
You can have a look at the docs - [https://mui.com/components/icons/](https://mui.com/components/icons/)

**Review by Devansu-Yadav — APPROVED** [review]
Like:
- UI for Homepage looks really cool!

Suggestion:-
- Please review the comments that I have made above to improve the code.
- The NavBar can be made more responsive by adding a Side Navigation Menu which would be displayed on Mobile Screens.

File: src/Pages/Homepage/Homepage.jsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+            <button className="cover-btn">Explore</button>
+        </div>
+        
+      </div>
+
+      <div className="secondary-cover-container">
+        <div>
+          <img
+            className="secondary-cover"
+            src={require("../../Assets/secondary-cover-1.jpg")}
```
Comment (OmKakatkar) [inline src/Pages/Homepage/Homepage.jsx:23]: You can use a media manager like cloudinary to host all the images and link the in the src

**Review by OmKakatkar — APPROVED** [review]
The folder structure is good
Like the use of CSS variables for colors

**Pull request merged**